cli/cmd/tables.go

```diff
@@ -34,6 +36,7 @@ func NewCmdTables() *cobra.Command {
 	}
 	cmd.Flags().String("output-dir", "cq-docs", "Base output directory for generated files")
 	cmd.Flags().String("format", "json", "Output format. One of: json, markdown")
+	cmd.Flags().String("filter", "all", "Filter tables. One of: all, spec")
```

Comment:
Maybe make default empty?

Reply:
I think `all` conveys better what would be the output the default behavior, WDYT?

Reply:
IMO `--filter` is a restrictive option, so, it it's empty/not specified, `allow all` is implied.
So I'd prefer saying that the default is `""`

Reply:
I think from a user perspective the behavior is the same. If they don't specify the flag it will return all tables. Also specifying `--filter all` seem nicer than? Otherwise it looks like a boolean flag I think